contrib_bots/lib/ConverterBot/docs.md

```diff
@@ -0,0 +1,69 @@
+# Converter bot
+
+This bot allows users to perform conversions for various measurement units.
+
+## Usage
+
+Run this bot as described in [here](http://zulip.readthedocs.io/en/latest/bots-guide.html#how-to-deploy-a-bot).
+
+Use this bot with the following command
+
+`@convert <number> <unit_from> <unit_to>`
+
+This will convert `number`, given in the unit `unit_from`, to the unit `unit_to`
+and print the result.
+
+* `number` can be any floating-point number, e.g. 12, 13.05, 0.002.
+* `unit_from` and `unit_to` are two units from [the following](#supported-units) table.
+* `unit_from` and `unit_to` can be preceded by [these](#supported-prefixes) prefixes.
+
+### Supported units
+
+| Physical quantity | units |
+| ----------------- | ----- |
+| Information | Bit, Byte |
+| Temperature | Celsius (c), Fahrenheit (f), Kelvin (k) |
+| Length | Foot (ft), Inch (in), Meter (m), Mile (mi), Nautical-Mile (nmi), Yard (y) |
+| Area | Acre, Are, Hectare (ha), Square-Centimeter (cm^2), Square-Foot (ft^2), Square-Inch (in^2), Square-Kilometer (km^2), Square-Meter (m^2), Square-Yard (y^2) |
```

Comment:
The order is alphabetical here, just as in the code. Should I change this?
Square kilometer and centimeter are not covered by the prefixes, since they are a unit for their own kilosquare-kilometer is covered, it would be 1000 square-kilometers. one square-kilometer is 1,000,000 square-meters, so the simple prefix-rule I use in the code does not work here.